UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -96,7 +96,7 @@ private BodyTypeName ThisBodyType
 		private List<ExternalCustomisation> ExternalCustomisationStorage = new();
 		private Dictionary<string, BodyPartCustomisationBase> OpenBodyCustomisation = new();
 
-		private List<PlayerHealthData> allSpecies;
+		public List<PlayerHealthData> AllSpecies;
```

Comment:
need to add formally serialised as

Reply:
scrap that

Reply:
Doesn't need to because it was never set via the inspector.